package-files/sshd_config

```diff
@@ -0,0 +1,5 @@
+Match group infra-users
```

Comment:
Not sure this should exist in the package. Might be better to template the file in code and write it out to `/etc/ssh/sshd_config.d/`

Reply:
We definitely shouldn't install it directly to `/etc/ssh/sshd_config.d/` in the package. I was thinking it could be installed to `/etc/infra/sshd_config` and the setup command could move it to `/etc/ssh/sshd_config.d/infra.conf`.

We could alternatively have the setup command write the file, but I think that might make it more difficult for someone to integrate with automation. By creating the file at `/etc/infra/sshd_config` the automation can just move the file into place.

What would we make a variable in this if we made it a template?  Are there other advantages to writing it out from the setup command?

Reply:
The location of `infra`, the group name though when I said template, I was referring broadly to writing it out to a file from code